eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -9,304 +9,359 @@ parameters:
+        parameters:
+          buildName: WindowsX64_Online_MsftSdk
+          isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+          vmrBranch: ${{ variables.VmrBranch }}
+          architecture: x64
+          pool:
+            name: ${{ variables.defaultPoolName }}
+            demands: ${{ variables.defaultPoolDemandsWindows }}
+          container: ''
+          excludeOmniSharpTests: true        # ✅
+          targetRid: 'win-x64'               # 📝
 
-    - template: ../jobs/vmr-build.yml
-      parameters:
-        # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-        buildName: Ubuntu2204Arm64_Offline_MsftSdk
-        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-        vmrBranch: ${{ variables.VmrBranch }}
-        architecture: arm64
-        pool: ${{ parameters.poolInternalArm64 }}
-        container: ${{ parameters.ubuntu2204ArmContainer }}
-        buildFromArchive: false            # 🚫
-        enablePoison: false                # 🚫
-        excludeOmniSharpTests: false       # 🚫
-        runOnline: false                   # 🚫
-        useMonoRuntime: false              # 🚫
-        withPreviousSDK: false             # 🚫
+      ### Additional jobs for full build ###
+      - ${{ if eq(parameters.isLiteBuild, false) }}:
 
-    - template: ../jobs/vmr-build.yml
-      parameters:
-        # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-        buildName: Fedora39_Offline_CurrentSourceBuiltSdk
-        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-        vmrBranch: ${{ variables.VmrBranch }}
-        architecture: x64
-        pool:
-          name: ${{ variables.defaultPoolName }}
-          demands: ${{ variables.defaultPoolDemands }}
-        container: ${{ parameters.fedora39Container }}
-        buildFromArchive: false            # 🚫
-        enablePoison: false                # 🚫
-        excludeOmniSharpTests: false       # 🚫
-        runOnline: false                   # 🚫
-        useMonoRuntime: false              # 🚫
-        withPreviousSDK: false             # 🚫
-        reuseBuildArtifactsFrom: Fedora39_Offline_MsftSdk
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            buildName: MarinerCrossArm64_Online_MsftSdk
+            isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+            vmrBranch: ${{ variables.VmrBranch }}
+            architecture: arm64
+            pool:
+              name: ${{ variables.defaultPoolName }}
+              demands: ${{ variables.defaultPoolDemandsLinux }}
+            container: ${{ variables.marinerArm64CrossContainer }}
+            excludeOmniSharpTests: true        # ✅
+            crossRootFs: '/crossrootfs/arm64'  # 📝
+            targetRid: 'linux-arm64'           # 📝
+
+        - template: ../jobs/vmr-build.yml
+          parameters:
+            buildName: OSXCrossArm64_Online_MsftSdk
```

Comment:
The MsftSdk suffix doesn't add value/make sense IMO in the vertical build context.  This is important in the source-build version because the legs build with different SDKs.  Likewise all vertical builds will be online won't they?  Removing this would simplify the build leg names.